darwin/future/core/types/common.py

```diff
@@ -83,3 +85,6 @@ def __init__(self, value: Dict[str, str]) -> None:
 
     def __str__(self) -> str:
         return "?" + "&".join(f"{k}={v}" for k, v in self.value.items())
+
```

Comment:
Useful for combining parameters in the core functions when required for pagination which I treat seperately